tools/m2-repo-cleaner/src/main/java/org/fusesource/ide/tools/mavenutils/M2RepoCleaner.java

```diff
@@ -101,4 +118,12 @@ private static void handleCorruptedZip(File file) {
 			}
 		}
 	}
+	
+	private static void handleLastUpdatedFile(File file) {
+		handleCorruptedZip(file);
+//		corruptedFiles.add(file.getPath());
```

Comment:
I thought that I provided a comment already but can't find it.

When does the .lastupdated file is created?
If it is often/on each check, the new behavior means that all artefacts are cleaned everytime and so we will fall into the corrupted jars issue and it will cause too many job failure